src/view/configuration/configuration.jsx

```diff
@@ -535,7 +535,7 @@ const Configuration = ({ formikProps, initInfo }) => {
                           component={EditorButton}
                           language="javascript"
                           placeholder={
-                            '// Modify content.xdm as necessary. For example:\n// content.xdm.product = "shirt";'
+                            '// Modify content.xdm as necessary. There is no need to wrap the code in a function\n// or return a value. For example:\n// content.xdm.product = "shirt";'
```

Comment:
Nice! 👍

Reply:
I know this is just an example, but content.xdm.product isn't a valid xdm field.  We could go with
```content.xdm.web.webPageDetails.name = "Checkout";```

Reply:
Done. Thanks!